app/main/views/briefs.py

```diff
@@ -118,6 +118,8 @@ def list_briefs():
 
     with_users = request.args.get('with_users', 'false').lower() == 'true'
 
```

Comment:
Not sure I'd bother with the space here because these lines do such a simila job, might be cool to group them.
I _certainly_ don't feel too strongly though.